src/hid/hid_device.cc

```diff
@@ -231,45 +226,51 @@ Status HidDevice::ExchangeCbor(Command command,
   if (1 + payload.size() > kMaxDataSize) return Status::kErrInvalidLength;
   std::vector<uint8_t> send_data = {static_cast<uint8_t>(command)};
   send_data.insert(send_data.end(), payload.begin(), payload.end());
+  return SendRecvCommand(kCtapHidCbor, send_data, response_cbor, true, true,
+                         false, expect_up_check);
+}
 
-  uint8_t cmd = kCtapHidCbor;
-  OK_OR_RETURN(SendCommand(cmd, send_data));
+Status HidDevice::SendCtapHid(const std::vector<uint8_t>& payload,
+                              std::vector<uint8_t>* response) const {
+  if (payload.empty() || payload.size() - 1 > kMaxDataSize) {
```

Comment:
Why restrict the payload length like that? We is it okay if it's just a little bit too long, but not in other cases?